action-api/src/test/java/com/powsybl/action/EqualsActionTest.java

```diff
@@ -0,0 +1,472 @@
+                .addEqualityGroup(action1, action2)
+                .addEqualityGroup(action3)
+                .addEqualityGroup(action4)
+                .addEqualityGroup(action5)
+                .testEquals();
+    }
+
+    @Test
+    void shuntCompensatorAction() {
+        ShuntCompensatorPositionAction action1 = new ShuntCompensatorPositionActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withSectionCount(10)
+                .build();
+        ShuntCompensatorPositionAction action2 = new ShuntCompensatorPositionActionBuilder().withId("id")
+                .withShuntCompensatorId("neId")
+                .withSectionCount(10)
+                .build();
+        ShuntCompensatorPositionAction action3 = new ShuntCompensatorPositionActionBuilder().withId("id2")
+                .withNetworkElementId("neId")
+                .withSectionCount(11)
+                .build();
+        ShuntCompensatorPositionAction action4 = new ShuntCompensatorPositionActionBuilder().withId("id")
+                .withNetworkElementId("neId2")
+                .withSectionCount(10)
+                .build();
+        ShuntCompensatorPositionAction action5 = new ShuntCompensatorPositionActionBuilder().withId("id2")
+                .withNetworkElementId("neId")
+                .withSectionCount(10)
+                .build();
+        new EqualsTester()
+                .addEqualityGroup(action1, action2)
+                .addEqualityGroup(action3)
+                .addEqualityGroup(action4)
+                .addEqualityGroup(action5)
+                .testEquals();
+    }
+
+    @Test
+    void staticVarCompensatorAction() {
+        Action action1 = new StaticVarCompensatorActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withReactivePowerSetpoint(10.)
+                .withVoltageSetpoint(5.)
+                .withRegulationMode(StaticVarCompensator.RegulationMode.VOLTAGE)
+                .build();
+        StaticVarCompensatorAction action2 = new StaticVarCompensatorActionBuilder().withId("id")
+                .withStaticVarCompensatorId("neId")
+                .withReactivePowerSetpoint(10.)
+                .withVoltageSetpoint(5.)
+                .withRegulationMode(StaticVarCompensator.RegulationMode.VOLTAGE)
+                .build();
+        Action action3 = new StaticVarCompensatorActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withReactivePowerSetpoint(10.)
+                .withRegulationMode(StaticVarCompensator.RegulationMode.VOLTAGE)
+                .build();
+        Action action4 = new StaticVarCompensatorActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withReactivePowerSetpoint(10.)
+                .withVoltageSetpoint(5.)
+                .build();
+        StaticVarCompensatorAction action5 = new StaticVarCompensatorActionBuilder().withId("id")
```

Comment:
To fully cover the `equals()` method, could you add an action with a change on the following attribute:
- staticVarCompensatorId